src/main/scala/io/viash/helpers/IO.scala

```diff
@@ -218,4 +218,18 @@ object IO {
     val it = Files.find(sourceDir, Integer.MAX_VALUE, (p, b) => filter(p, b)).iterator()
     it.asScala.toList
   }
+
+  /**
+    * 
+    */
+  def preResolveProject(path: String, projectURI: Option[URI]): (String, URI) = {
+    if (projectURI.isEmpty) {
+      throw new RuntimeException(s"One of the resources is relative to the project root ($path), but no project config file (_viash.yaml) could be found.")
+    }
+    (path.replaceFirst("/", ""), projectURI.get)
```

Comment:
Replace function so that only the first character can be replaced or double check that the function is only called when the first character is '/'?

Reply:
Added assertion, thanks!